symphony/lib/toolkit/class.emailgateway.php

```diff
@@ -630,7 +632,14 @@ protected function getSectionAttachments()
 
             if ($file_content !== false && !empty($file_content)) {
                 $output .= $this->boundaryDelimiterLine('multipart/mixed')
-                     . $this->contentInfoString($file['mime-type'], $file['file'], $file['filename'], $file['charset'])
+                     . $this->contentInfoString(
+                            isset($file['mime-type']) ? $file['mime-type'] : null,
+                            $file['file'],
+                            isset($file['filename']) ? $file['filename'] : null
+                            isset($file['charset']) ? $file['charset'] : null
+                            isset($file['cid']) ? $file['cid'] : null
```

Comment:
The three lines above are missing a trailing comma. PHP doesn't like this, really.

Reply:
;) I really wonder why my "lint dirty php" did not caught this... Thanks!

Reply:
See 5c759921ae